controller/TaskQueueWebApi.php

```diff
@@ -129,7 +131,49 @@ public function get()
     }
 
     /**
-     * @throws \common_exception_NotImplemented
+     * Gets the full url for a resource.
+     *
+     * @throws \Exception
+     */
+    public function getUrl()
```

Comment:
As part of the task description we are already sending over: id, taskname, status, masterstatus, tasklabel, createdad createdatelapsed, updated, updatedAt, category, hasfile. Why do we do an extra request for the URL?

Reply:
Good point. Actually, it is not really necessary to have a separated endpoint for the url.